app/models/card.rb

```diff
@@ -4,20 +4,29 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :review_date,
             presence: { message: 'Необходимо заполнить поле.' }
 
-  protected
-    def set_review_date
-      self.review_date = Time.now + 3.days
-    end
+  scope :review_card, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.